docs/FormatsAndConventions.md

```diff
@@ -138,6 +138,25 @@ e.g. `Error alert email has very long subject #5958`
 > to compensate for GitHub's lack of strong linking between the two.
 > Assume there is an invisible prefix in front of the PR title `Fixes issue : ...`
 
+## References to code elements
+
+Follow these conventions when referring to code elements e.g. when referring to a function name from a commit message.
+The objective is to be as concise as possible without causing ambiguities. Therefore, omit optional details when
+those details are not pertinent to the context.
+
+### Java
+
+* Variables: `package.class#variable` (optional: `package`)
+* Methods: `package.class#method(paramTypes):returnType` (optional: `package`, `paramTypes`, `:returnType`)
+
+e.g.
+* `Person#name`
+* `Person#getName()` 
+* `Person#getName():String` 
+* `Person#setName()` -- parameters omitted because only one `setName(String)` in the `Person` class
```

Comment:
>The body can be as long as we want although 72 char limit makes it awkward to have long method signatures in it.

Cases like `ArgumentTokenizer#extractArgumentValue(String, PrefixPosition, PrefixPosition)`, which has 78 characters in it, poses a problem with the 72 character limit. We could of course have this as an exception to the wrap at 72 characters guideline.

> Use of ellipsis adds 3 extra chars but avoids confusion between 'omitting parameters' and 'no parameters'

I am strongly in favor of this idea, and it doesn't add too many characters even in commit message titles.